README.rst

```diff
@@ -1,54 +1,38 @@
-********
-`SunPy`_
-********
+*********
+``sunpy``
+*********
 
-|Latest Version| |codecov| |matrix| |Research software impact| |DOI| |Powered by NumFOCUS|
+|Latest Version| |DOI| |element IM| |codecov| |Binder| |Powered by NumFOCUS|
 
 .. |Latest Version| image:: https://img.shields.io/pypi/v/sunpy.svg
    :target: https://pypi.python.org/pypi/sunpy/
-.. |matrix| image:: https://img.shields.io/matrix/sunpy:openastronomy.org.svg?colorB=%23FE7900&label=Chat&logo=matrix&server_fqdn=openastronomy.modular.im
+.. |DOI| image:: https://zenodo.org/badge/2165383.svg
+   :target: https://zenodo.org/badge/latestdoi/2165383
+.. |element IM| image:: https://img.shields.io/matrix/sunpy:openastronomy.org.svg?colorB=%23FE7900&label=Chat&logo=matrix&server_fqdn=openastronomy.modular.im
    :target: https://openastronomy.element.io/#/room/#sunpy:openastronomy.org
 .. |codecov| image:: https://codecov.io/gh/sunpy/sunpy/branch/main/graph/badge.svg
    :target: https://codecov.io/gh/sunpy/sunpy
-.. |Research software impact| image:: http://depsy.org/api/package/pypi/sunpy/badge.svg
-   :target: http://depsy.org/package/python/sunpy
-.. |DOI| image:: https://zenodo.org/badge/2165383.svg
-   :target: https://zenodo.org/badge/latestdoi/2165383
-.. |Powered by NumFOCUS| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
-   :target: https://numfocus.org
 .. |Binder| image:: https://mybinder.org/badge_logo.svg
    :target: https://mybinder.org/v2/gh/sunpy/sunpy/main?filepath=examples
+.. |Powered by NumFOCUS| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
+   :target: https://numfocus.org
 
-SunPy is an open-source Python library for Solar Physics data analysis and visualization.
-Our homepage `SunPy`_ has more information about the project.
-
-For some examples of using SunPy see our `gallery`_, to see the latest changes in SunPy see our `Changelog`_.
+``sunpy`` is the core library of the `SunPy Project <https://sunpy.org/>`__, a Python software package which is designed to provide the fundamental tools for accessing, loading and interacting with solar physics data in Python.
```

Comment:
The object of this sentence now feels ambiguous between the library and the project to me now.